manager/src/main/java/com/bakdata/quick/manager/graphql/GraphQLToProtobufConverter.java

```diff
@@ -0,0 +1,326 @@
+     * @param fileBuilder The Protobuf FileDescriptor builder that contains the converted messages.
+     */
+    private static void generateFile(
+        final String messageName,
+        final List<GraphQLFieldDefinition> fieldDefinitions,
+        final FileDescriptorProto.Builder fileBuilder) {
+
+        final Builder currentMessage = DescriptorProto.newBuilder().setName(messageName);
+
+        for (int index = 0; index < fieldDefinitions.size(); index++) {
+
+            final GraphQLFieldDefinition graphQLFieldDefinition = fieldDefinitions.get(index);
+            final GraphQLOutputType graphQLType = graphQLFieldDefinition.getType();
+
+            final Label label = isNonNull(graphQLType) ? Label.LABEL_REQUIRED : Label.LABEL_OPTIONAL;
+            final GraphQLType unwrappedType = isNonNull(graphQLType) ? unwrapOne(graphQLType) : graphQLType;
+
+            createMessage(fileBuilder,
+                currentMessage,
+                graphQLFieldDefinition,
+                unwrappedType,
+                index + 1,
+                label);
+        }
+
+        if (!fileBuilder.getMessageTypeList().contains(currentMessage.build())) {
+            fileBuilder.addMessageType(currentMessage);
+        }
+    }
+
+    private static void createMessage(
+        final FileDescriptorProto.Builder fileBuilder,
+        final Builder currentMessage,
+        final GraphQLFieldDefinition graphQLFieldDefinition,
+        final GraphQLType graphQLType,
+        final int fieldNumber,
+        final Label label) {
+        if (graphQLType instanceof GraphQLObjectType) {
+            handleObjectType(fileBuilder, currentMessage, graphQLFieldDefinition, fieldNumber, label,
+                (GraphQLObjectType) graphQLType);
+
+        } else if (isScalar(graphQLType)) {
+            final FieldDescriptorProto scalarFieldBuilder =
+                createFieldDescriptor((GraphQLScalarType) graphQLType,
+                    graphQLFieldDefinition.getName(),
+                    fieldNumber,
+                    label);
+            currentMessage.addField(scalarFieldBuilder);
+
+        } else if (isEnum(graphQLType)) {
+            handleEnumType(fileBuilder, currentMessage, graphQLFieldDefinition, fieldNumber, label,
+                (GraphQLEnumType) graphQLType);
+        } else if (isList(graphQLType)) {
+            final GraphQLList graphQLList = (GraphQLList) graphQLType;
+
+            createMessage(fileBuilder,
+                currentMessage,
+                graphQLFieldDefinition,
+                graphQLList.getWrappedType(),
+                fieldNumber,
+                Label.LABEL_REPEATED);
+        } else {
+            throw new BadArgumentException(
+                String.format("Type %s not recognized", GraphQLTypeUtil.simplePrint(graphQLType)));
+        }
+    }
+
+    private static void handleObjectType(
+        final FileDescriptorProto.Builder fileBuilder,
+        final Builder currentMessage,
```

Comment:
```suggestion
        final Descriptor.Builder currentMessage,
```
As a parameter, `Builder` is to generic

Reply:
This is not resolved yet

Reply:
This is not resolved yet